test/integration/scheduler/preemption/preemption_test.go

```diff
@@ -123,7 +124,10 @@ func (fp *tokenFilter) Filter(ctx context.Context, state *framework.CycleState,
 }
 
 func (fp *tokenFilter) PreFilter(ctx context.Context, state *framework.CycleState, pod *v1.Pod) (*framework.PreFilterResult, *framework.Status) {
-	return nil, nil
+	if !fp.EnablePreFilter || fp.Tokens > 0 {
+		return nil, nil
+	}
+	return nil, framework.NewStatus(framework.Unschedulable)
```

Comment:
If this function is never returning Skip, how is it related to the issue?

Reply:
Because the testing scheduling profile is atop default one, and default profile includes 3 plugin that return Skip (VolumeRestrictions, PodTopologySpread and InterPodAffinity).

Reply:
Please add a comment in the subtest description about the intention of the test and why it works

Reply:
added.